pkg/backend/display/progress.go

```diff
@@ -587,15 +587,10 @@ func (display *ProgressDisplay) refreshAllRowsIfInTerminal() {
 		var maxColumnLengths []int
 		display.convertNodesToRows(rootNodes, maxSuffixLength, &rows, &maxColumnLengths)
 
-		for i, row := range rows {
-			var id string
-			if i == 0 {
-				id = "#"
```

Comment:
this was old code from back when we printed a number at the start of the line.  we don't do that anymore, so there's no need to use # as the ID as '0' is fine here.